dist/ramda.js

```diff
@@ -1,3 +1,8 @@
+//  Ramda v0.9.1
+//  https://github.com/ramda/ramda
+//  (c) 2013-2015 Scott Sauyet and Michael Hurley
+//  Ramda may be freely distributed under the MIT license.
+
```

Comment:
Sorry, have been bed-bound for days without good internet.  (Going back to doctor in an hour.)  I finally realized that I could type with a real keyboard by using my phone as a hotspot.  (should have thought of that ages ago.)  But could only connect for some reason from old Windows laptop.  We don't seem to have a full build cycle working on Windows.  Either that or I messed it up.